app/controllers/campaigns_controller.rb

```diff
@@ -5,10 +5,13 @@ class CampaignsController < ApplicationController
   before_action :show_navbar_campaign
 
   def index
+    @company_tags = Category
+                      .where(company_id: current_user.company_id)
+                      .pluck(:title)
+                      .uniq
```

Comment:
@harold93 .uniq is a ruby method that's going to iterate over the ActiveRecord Collection output by :pluck.
Use .distinct instead, that is going to charge postgress with the unicity contraint, and also @company_tags will be an ActiveRecord collection instead of an array